compute/resource_cluster.go

```diff
@@ -107,6 +107,18 @@ func resourceClusterSchema() map[string]*schema.Schema {
 			Optional: true,
 			Computed: true,
 		}
+		s["is_pinned"] = &schema.Schema{
+			Type:     schema.TypeBool,
+			Optional: true,
+			Default:  false,
+			DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
can we add a comment on why this diffsuppressfunc is needed @alexott afaik this should not be required right?

Reply:
ahh i see the problem now there is no get pinned or get cluster does not yield pinned status.